lib/xbox/emu_socket.cpp

```diff
@@ -0,0 +1,623 @@
+/*
+ *      Copyright (C) 2005-2013 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
```

Comment:
This is a showstopper. This file is GPLv2 while libsmb2 is LGPLv2.1.
This change would change the licence of libsmb2 and is thus a showstopper.

We would need to ask the kodi folks to re-licence this file as LGPL before we can use it.
Or we would have to re-write it.

There are I think two additional files that are GPLv2 too that needs to be re-licenced or replaced.

Reply:
Well like i said above i have to merge these guys but using LGPLv2.1 code

Reply:
Seeing here some code are not GPL :+1:  At least but i will probally Join these non GPL code into one and use mvcs gethost instead of xbmc ones.

Reply:
Removed the XBMC socket emulation and using wide code instead.